lkvm stage1 doesn't build

**Environment**

```
rkt Version: 1.18.0
appc Version: 0.8.8
Go Version: go1.7.3
Go OS/Arch: linux/amd64
Features: -TPM +SDJOURNAL
--
Linux 4.8.2-1-userns x86_64
--
NAME="Arch Linux"
ID=arch
PRETTY_NAME="Arch Linux"
ANSI_COLOR="0;36"
HOME_URL="https://www.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://bugs.archlinux.org/"
--
systemd 231
+PAM -AUDIT -SELINUX -IMA -APPARMOR +SMACK -SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD +IDN
```

**What did you do?**

```
$ git diff
diff --git a/scripts/build-rir.sh b/scripts/build-rir.sh
index d2406fb..75bad14 100755
--- a/scripts/build-rir.sh
+++ b/scripts/build-rir.sh
@@ -4,7 +4,7 @@ set -xe
 
 SRC_DIR="${SRC_DIR:-$PWD}"
 BUILDDIR="${BUILDDIR:-$PWD/build-rir}"
-RKT_BUILDER_ACI="${RKT_BUILDER_ACI:-coreos.com/rkt/builder:1.1.0}"
+RKT_BUILDER_ACI="${RKT_BUILDER_ACI:-coreos.com/rkt/builder:1.1.1}"
 RKT_CMD="${RKT_CMD:-rkt}"
 
 mkdir -p $BUILDDIR

$ export BUILDDIR=$PWD/release-build && mkdir -p $BUILDDIR && sudo BUILDDIR=$BUILDDIR ./scripts/build-rir.sh
```

**What did you expect to see?**

A working build.

**What did you see instead?**

```
...
  BUILD EXT    lkvm
  ACTOOL       /opt/build-rkt/target/bin/stage1-fly.aci
Makefile:311: Skipping optional libraries: bfd GTK3 vncserver SDL zlib aio
KVMTOOLS_VERSION = 3.18.0
Makefile:311: Skipping optional libraries: bfd GTK3 vncserver SDL zlib aio
  WGET         https://alpha.release.core-os.net/amd64-usr/1192.0.0/coreos_production_pxe_image.cpio.gz.sig => /opt/build-rkt/tmp/coreos-common/pxe.img.95ed5e4b2b836654a38d0a292f2592cb.sig
  EXTRACT      /opt/build-rkt/tmp/coreos-common/pxe.img => /opt/build-rkt/tmp/coreos-common/usr.squashfs
/usr/bin/ld: /tmp/cc6rnpWm.o: relocation R_X86_64_32S against `.data' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: Nonrepresentable section on output
collect2: error: ld returned 1 exit status
make[1]: *** [guest/pre_init] Error 1
make[1]: *** Waiting for unfinished jobs....
  UNSQUASHFS   /opt/build-rkt/tmp/coreos-common/usr.squashfs => /opt/build-rkt/tmp/usr_from_coreos/cbu/rootfs/usr
  UNSQUASHFS   /opt/build-rkt/tmp/coreos-common/usr.squashfs => /opt/build-rkt/tmp/usr_from_kvm/cbu/rootfs/usr
stage1/usr_from_kvm/lkvm.mk:30: recipe for target '/opt/build-rkt/stamps/__stage1_usr_from_kvm_lkvm_mk_build.stamp' failed
make: *** [/opt/build-rkt/stamps/__stage1_usr_from_kvm_lkvm_mk_build.stamp] Error 2
make: *** Waiting for unfinished jobs....
```

I did a at first sight innocent version bump in rkt-builder from [1.1.0](https://github.com/coreos/rkt-builder/releases/tag/v1.1.0) to [1.1.1](https://github.com/coreos/rkt-builder/releases/tag/v1.1.1) but unfortunately it seems that the lkvm build stopped working.

I don't see huge changes in the gcc compiler chain though:

rkt-builder v1.1.1:
binutils 2.27-9+b1
...
gcc 4:6.1.1-1
gcc-5-base 5.4.1-3
gcc-6 6.2.0-10
gcc-6-base 6.2.0-10

rkt-builder v1.1.0:
binutils 2.27-8
...
gcc 4:6.1.1-1
gcc-4.9-base 4.9.4-2
gcc-5-base 5.4.1-2
gcc-6 6.2.0-3
gcc-6-base 6.2.0-3

@coreos/rkt-kvm-maintainers do you mind to have a look this?

--- a/stage1/usr_from_kvm/kernel/patches/0002-for-debian-gcc.patch
+++ b/stage1/usr_from_kvm/kernel/patches/0002-for-debian-gcc.patch
@@ -1,0 +1,13 @@
+diff --git a/Makefile b/Makefile
+index b249529..bedd4ab 100644
+--- a/Makefile
++++ b/Makefile
+@@ -346,7 +346,7 @@ include scripts/Kbuild.include
+ # Make variables (CC, etc...)
+ AS		= $(CROSS_COMPILE)as
+ LD		= $(CROSS_COMPILE)ld
+-CC		= $(CROSS_COMPILE)gcc
++CC		= $(CROSS_COMPILE)gcc -no-pie -fno-pic
+ CPP		= $(CC) -E
+ AR		= $(CROSS_COMPILE)ar
+ NM		= $(CROSS_COMPILE)nm

--- a/stage1/usr_from_kvm/lkvm.mk
+++ b/stage1/usr_from_kvm/lkvm.mk
@@ -5,7 +5,7 @@ LKVM_BINARY := $(LKVM_SRCDIR)/lkvm-static
 LKVM_ACI_BINARY := $(HV_ACIROOTFSDIR)/lkvm
 LKVM_GIT := https://kernel.googlesource.com/pub/scm/linux/kernel/git/will/kvmtool
 # just last published version (for reproducible builds), not for any other reason
-LKVM_VERSION := d62653e177597251c24494a6dda60acd6d846671
+LKVM_VERSION := 1cd6f516264ad2ad83fad3dc1264d6ff4bcd17b2
 
 LKVM_STUFFDIR := $(MK_SRCDIR)/lkvm
 LKVM_PATCHESDIR := $(LKVM_STUFFDIR)/patches

